brightray/browser/browser_main_parts.cc

```diff
@@ -55,6 +58,8 @@
 
 namespace brightray {
 
+void OverrideMacAppLogsPath();
```

Comment:
Add a comment about the Mac logs file being somewhere else because of Objective-C implementation.